fedbiomed/common/message.py

```diff
@@ -925,7 +931,6 @@ class FAReply(RequestReply, RequiresProtocolVersion):
     node_id: str
     node_name: str
```

Comment:
As it also commented in `FAjob` class, I think it could be nice to add `analytics_type` to the `FAReply`.

```suggestion
    node_id: str
    node_name: str
    analytics_type: str
```

Reply:
This is not going to be very useful for this version but it will be for once we add more analytics types such as st deviation, histogram etc.